packages/rpc-graphql/README.md

```diff
@@ -3,61 +3,1160 @@
+            }
+        }
+    } = account;
+    if (accountType === "mint") {
+      acc.mints.push(accountParsedInfo);
+    } else {
+      acc.tokenAccounts.push(accountParsedInfo);
+    }
+    return acc;
+  },
+  { mints: [], tokenAccounts: [] }
+);
+```
+
+Account data encoding in `base58`, `base64`, and `base64+zstd` is also
+supported, as well as `dataSlice` and `filter`!
+
+```typescript
+const source = `
+    query myQuery(
+        $programAddress: String!,
+        $commitment: Commitment,
+        $dataSlice: DataSlice,
+        $encoding: AccountEncoding,
+    ) {
+        programAccounts(
+            programAddress: $programAddress,
+            commitment: $commitment,
+            dataSlice: $dataSlice,
+            encoding: $encoding,
+        ) {
+            ... on AccountBase64 {
+                data
+            }
+        }
+    }
+`;
+
+const variableValues = {
+    programAddress: 'DXngmJfjurhnAwbMPgpUGPH6qNvetCKRJ6PiD4ag4PTj',
+    commitment: 'confirmed',
+    dataSlice: {
+        length: 5,
+        offset: 0,
+    },
+    encoding: 'base64',
+};
+
+const result = await rpcGraphQL.query(source, variableValues);
+
+expect(result).toMatchObject({
+    data: {
+        programAccounts: expect.arrayContaining([
+            {
+                data: 'dGVzdCA=',
+            },
+        ]),
+    },
+});
+```
+
+### Nested Program Account Queries
+
+Querying program accounts and applying nested queries to the objects within the
+response list is an area where RPC-GraphQL really shines.
+
+Consider an example where we want to get the **sum** of every lamports balance
+of every **owner of the owner** of each token account, while discarding any
+mint accounts.
+
+```typescript
+const source = `
+    query getLamportsOfOwnersOfOwnersOfTokenAccounts {
+        programAccounts(programAddress: "TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA") {
+            ... on TokenAccount {
+                data {
+                    parsed {
+                        info {
+                            owner {
+                                owner {
+                                    lamports
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+`;
+
+const result = await rpcGraphQL.query(source);
+
+const sumOfAllLamportsOfOwnersOfOwnersOfTokenAccounts = result
+    .map(o => o.account.data.parsed.info.owner.owner.lamports)
+    .reduce((acc, lamports) => acc + lamports, 0);
+```
+
+## Querying Transactions
+
+The `Transaction` interface contains common fields across all transactions.
+
+`src/schema/transaction/types.ts: TransactionInterface`
+
+```graphql
+interface Transaction {
+    blocktime: BigInt,
+    encoding: String,
+    meta: TransactionMeta, # Interface
+    slot: BigInt,
+}
+```
+
+Notice the `TransactionMetaInterface` type present in the transaction
+interface.
+
+```graphql
+interface TransactionMeta {
```

Comment:
Nitpick: a few of these `interface` blocks are marked as graphql code blocks. Should they be TS? In graphite at least it doesn't render them as TS

Reply:
Yes these are GraphQL interface schemas!

Reply:
Gotcha! They look good in github too, nice!